Prefer unix chrono sys impl on wasm32-linux (avoid duplicate mod inner).

--- a/third_party/rust/chrono/src/sys.rs
+++ b/third_party/rust/chrono/src/sys.rs
@@ -16,7 +16,7 @@
 
 use std::time::{SystemTime, UNIX_EPOCH};
 
-#[cfg(any(target_arch = "wasm32", target_env = "sgx"))]
+#[cfg(any(all(target_arch = "wasm32", not(target_os = "linux")), target_env = "sgx"))]
 #[path = "sys/stub.rs"]
 mod inner;
 
